more_itertools/more.py

```diff
@@ -2152,10 +2150,11 @@ class numeric_range(abc.Sequence, abc.Hashable):
 
     """
 
-    _EMPTY_HASH = hash(range(0, 0))
+    _EMPTY_HASH = hash(range(0))
```

Comment:
This edit is okay, but arguably `range(0, 0)` is clearer that this is an empty range.

Mostly, we should defer to the intensions of the person who wrote this could rather than blindly making substitutions based on ruff recommendations which don't actually know the author's intent.